dev/how-to/upgrade/tidb-binlog.md

```diff
@@ -1,29 +1,30 @@
 ---
-title: TiDB-Binlog Cluster Upgrade
-summary: Learn how to upgrade the cluster version of TiDB-Binlog.
+title: TiDB Binlog Cluster Upgrade
+summary: Learn how to upgrade the cluster version of TiDB Binlog.
 category: reference
+
 ---
 
-# TiDB-Binlog Cluster Upgrade
+# TiDB Binlog Cluster Upgrade
```

Comment:
Use `Upgrade TiDB Binlog Cluster` to keep it consistent with other titles in the same directory of TOC.